app/js/app.js

```diff
@@ -17,12 +17,10 @@ angular.module('app', [
 ]).config(function(RestangularProvider) {
 	RestangularProvider.setBaseUrl('https://api.tnyu.org/v2');
 
-	var ApiKey = "";

```

Comment:
:+1: to this! Definitely will find a better way to make admin-test work going forward :)